<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaUsuarios</class>
 <widget class="QDialog" name="VentanaUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="1">
    <widget class="QTableWidget" name="tabla"/>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="botAgregar">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botEliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Buscar por:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_dni">
       <property name="text">
        <string>DNI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_Nombre">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_Apellido">
       <property name="text">
        <string>Apellido</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="botton_usuario">
       <property name="text">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineBuscar"/>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="botCerrar">
     <property name="text">
      <string>Cerrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
